### ISSUE TITLE
Improve Operation usage documentation

### ISSUE THREAD

**Issue by pattyland** [hyperion-project/hyperion.docs#14]
Hi,
I would like to improve the documentation of Hyperion in the "Operation" section, but I am missing virtually all information about it.
While the installation and configuration is well described, virtually all information on usage is missing:
- Where is the installation of Hyperion?
- How do I start/stop/restart Hyperion?
- How do I get logs files?
etc
Do you guys see it the same way?
Then I would start collecting this info together and submit a PR.
LG Sören

**Comment by nvtkaszpir** [comment]
1. hyperion is usually installed on operating system, which has devices used for video processing (directly via framebuffer or something like USB device)
2. if you install it from system packages, then you should use system specific service manager, for example under Ubuntu 20.04 it would be systemd, which uses `sudo systemctl start hyperion` (or replace `start` with `stop`
3. this depends on installation, if you use systemd you may use journalctl or syslog to see the logs. Refer to those tools how to use them (usually `journalctl -u hyperion` should be sufficient with modern operating systems)

**Comment by pattyland** [comment]
@nvtkaszpir I'm sorry that you have now taken the trouble to formulate a good answer to my stupidly worded question.... I wrote this query after I installed HyperBian, there the documentation just stops after the installation part.
There is no "normally" and "if" but it should be clearly defined what is where: https://docs.hyperion-project.org/en/user/HyperBian.html#requirements

**Comment by nvtkaszpir** [comment]
@pattyland yes you are right, I believe docs are lacking in that matter, I believe `Installation` section needs `Running Hyperion` section.
Now there is only `Install -> Requirements/Install/.../Uninstall` and then `Configuration`.

Looks like `Run Hyperion` is completely missing but is quite crucial, but it varies depending on the running operating system (kind of standalone in CLI, as service (systemd/initd and so on), not to mention that for example  there is no explanaition between hyperiond or hyperion-x11 and other binaries, which is quite important.
I believe it was just left by accident because sometimes things are so obvious for experienced users that people forget about the inexperienced/newcomers :)

Any thoughts about structurixing that docs section, @Lord-Grey ?
